src/values/yes-no-value.js

```diff
@@ -34,7 +34,7 @@ export default class YesNoValue extends TextualValue {
       case this.isNeutral:
         return this.element.neutralChoice.label;
       default:
-        return this.textValue;
+        return this.textValue || '';
```

Comment:
this was undefined previously when the value was null. Chose to follow the text-field implementation and return an empty string.